iOS/Flows/Conversation/_New/ConversationViewController.swift

```diff
@@ -118,33 +114,32 @@ class ConversationViewController: FullScreenViewController,
         guard let controller = self.conversationController else { return }
 
         // Make sure messages are loaded before initializing the data.
-        if let mostRecentMessage = controller.messages.last {
+        if let mostRecentMessage = controller.messages.first {
```

Comment:
The first message in the array is now the most recent because of the message ordering change above.